devserver.sh

```diff
@@ -1,23 +1,39 @@
 #!/bin/sh
 
-work_dir=$(mktemp -d)
+work_dir="$(mktemp -d)"
 
-if [ -z $MVN ]; then
-    MVN=$(which mvn)
+function cleanup() {
+    podman stop cryostat-devserver-reports
+    rm -rf "${work_dir}"
+}
+trap cleanup EXIT
+
+if [ -z "${MVN}" ]; then
+    MVN="$(which mvn)"
 fi
 
 for i in archive clientlib conf templates probes; do
-    if [ -e $work_dir/$i ]; then
-        if [ ! -d $work_dir/$i ]; then
-            echo "$work_dir/$i already exists but is not a directory"
+    if [ -e "${work_dir}/${i}" ]; then
+        if [ ! -d "${work_dir}/${i}" ]; then
+            echo "${work_dir}/${i} already exists but is not a directory"
             exit 1
         fi
     else
-        mkdir $work_dir/$i
+        mkdir "${work_dir}/${i}"
     fi
 done
 
+if [ -z "$CRYOSTAT_CORS_ORIGIN" ]; then
+    "${MVN}" -DskipTests=true prepare-package
+else
+    "${MVN}" -DskipTests=true -Dcryostat.minimal=true prepare-package
+fi
+
+# HACK. The vertx-maven-plugin doesn't include target/assets on the classpath, so copy its contents into target/classes which is included
+cp -r target/assets/app/resources/* target/classes
+
 flags=(
+    "-cp 'target/classes:target/assets/app/resources:@target/dependency-plugin-classpath-file'"
```

Comment:
Since we already copied the contents of `target/assets/app/resources/` to `target/classes` on line 33, why do we need to specify `target/assets/app/resources/` in the JVM classpath? Is it because the `vertx-maven-plugin` classpath configuration is only concerned with the proper building of our application, while the JVM classpath has to do with running the built application (i.e. their respective classpaths fulfill different purposes)?

Reply:
Whoops. I think this line is actually no longer needed - I was experimenting with different ways to set the classpath for the main JVM and for the forked Vert.x JVM before, but eventually just settled on the "cp -r" hack instead. I'll remove this.